modules/nf-core/s4pred/runmodel/meta.yml

```diff
@@ -0,0 +1,47 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "s4pred_runmodel"
+description: Prediction of a protein's secondary structure from its amino acid sequence
+keywords:
+  - protein
+  - secondary structure
+  - prediction
+tools:
+  - "s4pred":
+      description: "Accurate prediction of a protein's secondary structure from its\
+        \ amino acid sequence"
+      homepage: "https://github.com/psipred/s4pred"
+      documentation: "https://github.com/psipred/s4pred"
+      tool_dev_url: "https://github.com/psipred/s4pred"
+      doi: "10.1093/bioinformatics/btab491"
+      licence: ["GPL v3-or-later"]
+      identifier: ""
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - fasta:
+        type: file
+        description: protein FASTA file containing one or more amino acid sequences to predict their respective secondary structures
+        pattern: "*.{fasta,fa,fas,fna,faa}"
```

Comment:
Could you add the ontology ?
https://edamontology.github.io/edam-browser/#format_1929

Reply:
Good idea, is there an example of another module that does this?

Reply:
Here is an example
https://nf-co.re/blog/2025/modules-ontology